src/librustdoc/html/static/main.js

```diff
@@ -142,8 +142,8 @@ function getSearchElement() {
     var TY_PRIMITIVE = itemTypes.indexOf("primitive");
     var TY_KEYWORD = itemTypes.indexOf("keyword");
 
-    onEachLazy(document.getElementsByClassName("js-only"), function(e) {
-        removeClass(e, "js-only");
+    onEachLazy([getSearchInput()], function(e) {
+        e.removeAttribute('disabled');
```

Comment:
Can you please move this block in the function `addSearchOptions`? I just realized that there is no point into enabling the search bar if the content wasn't loaded yet.